popup.js

```diff
@@ -52,7 +58,9 @@ previouses.forEach((previous) => {
 });
 
 async function setShortcuts() {
+  
   const extensionShortcuts = await chrome.commands.getAll();
+  
```

Comment:
There should be an error handling mechanism to handle if the 'getAll()' command fails.